src/base/QXmppJingleIq.cpp

```diff
@@ -461,8 +461,7 @@ void QXmppJingleIq::Content::toXml(QXmlStreamWriter *writer) const
 bool QXmppJingleIq::Content::parseSdp(const QString &sdp)
 {
     QList<QXmppJinglePayloadType> payloads;
-    QString line;
-    for (auto &line : sdp.split('\n')) {
+    for (auto &line : sdp.split(QChar(u'\n'))) {
```

Comment:
Isn't this also iterating over a temporary?

Reply:
This shall be fine, the temporary will live long enough, and the container shouldn't be detached since its refcount will be `1`.